app/views/events/_form.html.erb

```diff
@@ -43,6 +43,20 @@
     </div>
     <%=f.error_span(:active) %>
   </div>
+  <div class="form-group">
+    <%= f.label :veranstalter, :class => 'control-label col-lg-2' %>
```

Comment:
should be english, to stay consistent with all the other code (+ all other occurences of german words)